hangman/lib/hangman/game.ex

```diff
@@ -0,0 +1,97 @@
+defmodule Game do
+  @moduledoc false
+
+  defmodule State do
+    defstruct game_state: :initializing, turns_left: 7, letters: [], used: [], last_guess: "", random_word: ""
+  end
+
+  @doc "Create new game with default state and random word"
+  def newGame do
+    %State{ random_word: Dictionary.WordList.random_word() }
```

Comment:
## -2 elegance

this is crying out to be

~~~ elixir
Dictionary.WordList.random_word() 
|> new_game()
~~~

It's both clearer and removes the duplication.